examples/golang/polynomials/README.md

```diff
@@ -0,0 +1,49 @@
+# ICICLE example: Polynomials in Golang
+
+`ICICLE` provides Golang bindings to CUDA-accelerated C++ implementation of [Polynomials](../../../docs/docs/icicle/polynomials/overview.md).
```

Comment:
Lets link to the deployed version

https://dev.ingonyama.com/icicle/polynomials/overview

Reply:
@jeremyfelder docusaurus renders the links to use a full path, so it's actually better to user relative path links when rendering locally